lib/Backend/Encoder.h

```diff
@@ -11,7 +11,7 @@
 ///
 ///---------------------------------------------------------------------------
 
-typedef JsUtil::List<NativeOffsetInlineeFramePair, ArenaAllocator> InlineeFrameMap;
+typedef JsUtil::List<NativeOffsetInlineeFramePair, ArenaAllocator> ArenaInlineeFrameMap;
```

Comment:
Is this simply to make the name more descriptive or was there some other intent behind this rename?

Reply:
There is one for Recycler that got moved to the global name space that conflicted with this one.